README.md

```diff
@@ -55,12 +55,22 @@ the future.
 
 ## Documentation
 
-This is the main project repository. In this
-repository, documentation regarding the [design](docs/design),
-[requirements](docs/requirements) and [specifications](docs/specifications)
-for the Zonemaster implementation are available.
+This is the main project repository. In this repository, documentation
+regarding the [design](docs/internal/design),
+[requirements](docs/internal/requirements) and
+[specifications](docs/public/specifications) for the Zonemaster implementation
+are available.
```

Comment:
I think this section should be expanded:
1. Some more text about each category of document.
2. Link to installations documents should be included.
4. If you select "requirements" you would probably expect requirements for running Zonemaster, but that is not what you get.
5. Installation and possibly test case specifications are probably what most people want first, and they should be high up.